Integrations/FortiGate/FortiGate.py

```diff
@@ -429,6 +467,76 @@ def create_firewall_service_request(service_name, tcp_range, udp_range):
     return response
 
 
+def ban_ip(ip_addresses_array, time_to_expire):
```

Comment:
no need to specify `array`, using plural form is enough
```suggestion
def ban_ip(ip_addresses, time_to_expire):
```

Reply:
Could you please explain further?
How could someone reading the code be sure if the function is getting (and thus, sending in the request) an array or a string of multiple IPs, without actually having to read through the API or going back to the ban_ip_command function calling ban_ip?